SpanishFilter/sections/specific.txt

```diff
@@ -11,6 +11,9 @@ jc.ne10.uol.com.br,internetpasoapaso.com,acidadeon.com,leouve.com.br,larazon.es,
 ! ##.ad
 laopiniondemurcia.es,larazon.es,dicio.com.br,band.uol.com.br,xataka.com,onyxplay.com,genbeta.com,vagalume.com.br,xatakandroid.com,abril.com.br##.ad
 !
+wpolityce.pl##.tile--ads
+wpolityce.pl##aside.publication__forum-ad-scrolled
+wpolityce.pl##.publication__ad-scrolled-container
```

Comment:
Actually, it's a Polish website. Please move rules to the AdGuard Base - foreign file - find comment `Polish` - add the rules below.

```
! https://github.com/AdguardTeam/AdguardFilters/issues/102534
wpolityce.pl##.tile--ads
wpolityce.pl##.publication__forum-ad-scrolled
wpolityce.pl##.publication__ad-scrolled-container
```